networkx/algorithms/chordal/chordal_alg.py

```diff
@@ -242,6 +244,47 @@ def chordal_graph_treewidth(G):
         max_clique = max(max_clique,len(clique))
     return max_clique - 1
 
+@not_implemented_for('directed','multigraph')
+def chordal_simplicial_vertex(G):
+    """Returns the simplicial vertex of the chordal graph G. 
+    
+    Parameters
+    ----------
+    G : graph
+      A NetworkX graph 
+    
+    Returns
+    -------
+    Simplicial vertex of graphs G is vertex v such that closed neighborhood of v is clique in G.
+    
+    Raises
+    ------
+    NetworkXError
+        The algorithm does not support DiGraph, MultiGraph and MultiDiGraph. 
```

Comment:
Really, this should say

    NetworkXNotIImplemented
        If ``G`` is directed or a multigraph.

    NetworkXError
        If the input graph is not chordal.